install/kubernetes/cilium/charts/agent/templates/daemonset.yaml

```diff
@@ -64,7 +64,11 @@ spec:
             - --brief
 {{- else }}
           httpGet:
+{{- if .Values.global.ipv4.enabled }}
```

Comment:
For dual stack, only ipv4 endpoint /healthz is used, so do we need to bring up both endpoints :)?

As per https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#httpgetaction-v1-core, can we just get rid of `host` attribute ?

> Host name to connect to, defaults to the pod IP. You probably want to set "Host" in httpHeaders instead.

Reply:
> For dual stack, only ipv4 endpoint /healthz is used, so do we need to bring up both endpoints :)?

We can't tell from the agent beforehand in which way users will be checking the endpoint (we have no access to the helm chart AFAIK), so the only way we have are the `--enable-ipv{4,6}` flags to decide which endpoints to bring up. These are already considered. Maybe we could change the agent options in the helm chart accordingly.

> As per https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#httpgetaction-v1-core, can we just get rid of `host` attribute ?
> 
> > Host name to connect to, defaults to the pod IP. You probably want to set "Host" in httpHeaders instead.

I'll look into that and send a follow-up PR if applicable.